foyer/forcefields/ff.xsd

```diff
@@ -52,6 +52,20 @@
       </xs:simpleContent>
     </xs:complexType>
   </xs:element>
+  <xs:element name="UreyBradley">
+    <xs:complexType>
+      <xs:simpleContent>
+        <xs:extension base="xs:string">
+          <xs:attribute type="xs:float" name="d" use="optional"/>
+          <xs:attribute type="xs:float" name="k" use="optional"/>
+          <xs:attribute type="xs:string" name="type1" use="optional"/>
```

Comment:
Do you know if OpenMM supports the use of "class" here in addition to "type"? I'm guessing they do, since this seems to be the case for all of the other force definitions. I'm sure they have this documented somewhere although I'm having trouble finding any information...

Reply:
I suspect so but would be good to confirm. If so, the schema should allow this.